Documentation/Books/Manual/Programs/Arangorestore/FastClusterRestore.md

```diff
@@ -0,0 +1,286 @@
+Please refer to
+[this](Examples.md#factors-affecting-speed-of-arangorestore-in-a-cluster)
+section for further context on the factors affecting restore speed when restoring
+using _arangorestore_ in a Cluster.
+{% endhint %}
+
+Step 1: Copy the _dump_ directory to all _Coordinators_
+-------------------------------------------------------
+
+The first step is to copy the directory that contains the _dump_ to all machines
+where _Coordinators_ are running.
+
+{% hint 'tip' %}
+This step is not strictly required as the backup can be restored over the
+network. However, if the restore is executed locally the restore speed is
+significantly improved.
+{% endhint %}
+
+Step 2: Restore collection structures
+-------------------------------------
+
+The collection structures have to be restored from exactly one _Coordinator_ (any
+_Coordinator_ can be used) with a command similar to the following one. Please add
+any additional needed option for your specific use case, e.g. `--create-database`
+if the database where you want to restore does not exist yet:
+
+```
+arangorestore
+  --server.endpoint <endpoint-of-a-coordinator>
+  --server.database <database-name>
+  --server.password <password>
+  --import-data false
+  --input-directory <dump-directory>
+```
+
+{% hint 'info' %}
+If you are using v3.3.22 or higher, or v3.4.2 or higher, please also add in the
+command above the option `--replication-factor 1`.
+{% hint 'tip' %}
+
+The option `--import-data false`  tells _arangorestore_ to restore only the
+collection structure and no data.
+
+Step 3: Set _Replication Factor_ to 1
+--------------------------------------
+
+{% hint 'info' %}
+This step is **not** needed if you are using v3.3.22 or higher or v3.4.2 or higher
+and you have used in the previous step the option `--replication-factor 1`.
+{% hint 'tip' %}
+
+To speed up restore, it is possible to set the _replication factor_ to 1 before
+importing any data. Run the following command from exactly one _Coordinator_ (any
+_Coordinator_ can be used):
+
+```
+echo 'db._collections().filter(function(c) { return c.name()[0] !== "_"; })
+.forEach(function(c) { print("collection:", c.name(), "replicationFactor:",
+c.properties().replicationFactor); c.properties({ replicationFactor: 1 }); });'
+| arangosh
+  --server.endpoint <endpoint-of-a-coordinator>
+  --server.database <database-name>
+  --server.username <user-name>
+  --server.password <password>
+```
+
+Step 4: Create parallel restore scripts
+---------------------------------------
+
+Now that the Cluster is prepared, the `parallelRestore` script will be used.
+
+Please create the below `parallelRestore` script in any of your _Coordinators_.
+
+When executed (see below for further details), this script will create other scripts
+that can be then copied and then executed on each _Coordinator_.
+
+
+```
+#!/bin/sh
+#
+# Version: 0.2
+#
+# Release Notes:
+# - v0.3: ....... TODO .... fixed a bug when _ is used in the collection name
```

Comment:
this is still an open bug, per Luigi.  @lservini or @neunhoef ideas on how to fix it (how can the script be modified)? thanks!